quasar/src/components/editor/QEditor.js

```diff
@@ -261,14 +261,13 @@ export default Vue.extend({
     },
 
     onKeydown (e) {
-      const key = e.keyCode
-
       if (!e.ctrlKey) {
         this.refreshToolbar()
         this.$q.platform.is.ie && this.$nextTick(this.onInput)
         return
       }
 
+      const key = e.keyCode
```

Comment:
Moved this declaration to after the "non-ctrlKey" for a probably insignificant speedup.